charts/external-secrets/README.md

```diff
@@ -0,0 +1,234 @@
+
+  ```bash
+  kubectl create secret -n external-secrets generic ssh-creds --from-file=./identity --from-file=./identity.pub --from-file=./known_hosts
+  ```
+
+- Edit values file to the secret ref and path in values.yaml for your secrets repository
+
+### Bootstrapping leaf cluster
+
+- To authenticate the secret store on a leaf cluster with key/secret creds you will need to create a [ClusterResourceSet](https://docs.gitops.weave.works/docs/cluster-management/getting-started/#automatically-install-a-cni-with-clusterresourcesets) having the AWS secret and it will be on the leaf cluster through the [bootstrapping process](https://docs.gitops.weave.works/docs/cluster-management/getting-started/#add-a-cluster-bootstrap-config)
+
+  ```bash
+  kubectl create secret generic aws-sm-crs-secret --from-literal access-key=$KEY --from-literal secret-access-key=$SECRET --type=addons.cluster.x-k8s.io/resource-set
+  ```
+
+- Cluster Resource secret to be bootstrapped in the leaf cluster under bootstrap. Make sure to add the cluster selector label `secretmanager: aws` under GitOpsCluster in the cluster template
+
+  ```yaml
+  apiVersion: addons.cluster.x-k8s.io/v1alpha3
+  kind: ClusterResourceSet
+  metadata:
+    name: awssm-crs
+    namespace: default
+  spec:
+    clusterSelector:
+      matchLabels:
+        secretmanager: aws
+    resources:
+    - kind: Secret
+      name: aws-sm-crs-secret
+  ```
+
+
+- To add the ssh creds to flux to be able to access private repository you will need to create a [ClusterResourceSet](https://docs.gitops.weave.works/docs/cluster-management/getting-started/#automatically-install-a-cni-with-clusterresourcesets) having the SSH Creds and it will be on the leaf cluster through the [bootstrapping process](https://docs.gitops.weave.works/docs/cluster-management/getting-started/#add-a-cluster-bootstrap-config)
+
+  ```bash
+  kubectl create secret generic ssh-creds-crs-secret --from-file=./identity --from-file=./identity.pub --from-file=./known_hosts --type=addons.cluster.x-k8s.io/resource-set
+  ```
+
+- Cluster Resource secret to be bootstrapped in the leaf cluster under bootstrap
+
+  ```yaml
+  apiVersion: addons.cluster.x-k8s.io/v1alpha3
+  kind: ClusterResourceSet
+  metadata:
+    name: ssh-creds
+    namespace: default
+  spec:
+    clusterSelector:
+      matchLabels:
+        secretmanager: aws
+    resources:
+    - kind: Secret
+      name: ssh-creds-crs-secret
+  ```
+
+### Notes on the creating the leaf cluster
+
+- It should have flux bootstrapped on it. using cluster bootstrap config and it should have labels matching the cluster template.
+
+- ClusterResourceSet has cluster selector labels to choose which cluster to be installed on and it should have labels matching the cluster template.
+
+Full guide to bootstrap leaf cluster with flux with template [here](https://www.notion.so/weaveworks/Guide-How-To-Secrets-Management-with-flux-ad91b52e3ba5415c97e2235ae394bf4f)
```

Comment:
we could change this to a public source this profile could be used by customers that might not have access to our notion (most common case). 

actually, it feels that we could point for all or pieces of this documentation to our user documentation

Reply:
added the bootstrap guide next to readme and refereed it here